Foundation/NSJSONSerialization.swift

```diff
@@ -202,8 +202,24 @@ open class JSONSerialization : NSObject {
     
     /* Create a JSON object from JSON data stream. The stream should be opened and configured. All other behavior of this method is the same as the JSONObjectWithData:options:error: method.
      */
-    open class func jsonObject(with stream: InputStream, options opt: ReadingOptions = []) throws -> AnyObject {
-        NSUnimplemented()
+    open class func jsonObject(with stream: InputStream, options opt: ReadingOptions = []) throws -> Any {
+        var data = Data()
+        guard stream.streamStatus == .open || stream.streamStatus == .reading else {
+            throw NSError(domain: NSCocoaErrorDomain, code: CocoaError.propertyListReadCorrupt.rawValue, userInfo: [
+                "NSDebugDescription" : "Stream is not available for reading."
+                ])
+        }
+        repeat {
+            var buffer = [UInt8](repeating: 0, count: 1024)
```

Comment:
I tried to use Data instead of the buffer here,  but got stuck at the point where I am supposed to send in the UnsafeMutablePointer and an Int of maxLength to read the contents of the stream:

var dataChunk = Data()
try dataChunk.withUnsafeMutableBytes {(bytes: UnsafeMutablePointer<UInt8>)->Void in
                bytesRead = stream.read(bytes, maxLength: ??)

Is there a way I could read the InputStream to Data without the buffer?